.github/workflows/test.yml

```diff
@@ -9,7 +9,7 @@ on:
   pull_request:
 env:
   GO_VERSION: '1.16'
-  GO_VERSION_LEGACY: '1.14'
+  GO_VERSION_LEGACY: '1.15'
```

Comment:
This is already part of https://github.com/cri-o/cri-o/pull/5109. It also looks like that we cannot build with go 1.15 either.